components/switch/themes/spectrum.css

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+
+@container (--system: spectrum) {
+    .spectrum-Switch--emphasized {
```

Comment:
After we get that `emphasized` selector in the `index.css` file, we should be able to remove this rule.

Reply:
I think we have to keep this selector in order to keep the different colors between Express and Spectrum for the emphasized version.  See my comment in https://github.com/adobe/spectrum-css/pull/1496#discussion_r954309715